std_msgs/Header header
float64 thrust_cmd          # status reading either 0 or 1


